hazelcast/src/main/java/com/hazelcast/concurrent/lock/operations/LocalLockCleanupOperation.java

```diff
@@ -35,6 +35,14 @@
 
     private final String uuid;
 
+    /**
+     * This constructor should not be used to obtain an instance of this class; it exists to fulfill IdentifiedDataSerializable
+     * coding conventions.
+     */
+    public LocalLockCleanupOperation() {
+        uuid = "";
```

Comment:
As discussed with Vassilis, it's probably better to have the getId/factoryId to throw since:

- if I subclass a "non-local" operation to make it local and do not override the getId/factoryI they will return "wrong" values.
- if it's a local class it may use the non-arg constructor for sth else